### PR TITLE
joystick_drivers: 2.2.0-1 in 'crystal/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by nuclearsandwich**
Increasing version of package(s) in repository `joystick_drivers` to `2.2.0-1`:

- upstream repository: https://github.com/ros2/joystick_drivers.git
- release repository: https://github.com/ros2-gbp/joystick_drivers-release.git
- distro file: `crystal/distribution.yaml`
- bloom version: `0.6.9`
- previous version for package: `null`

### CODE REVIEW

File: crystal/distribution.yaml [hunk 1]
```diff
@@ -396,6 +396,24 @@ repositories:
       url: https://github.com/ros-perception/image_common.git
       version: ros2
     status: maintained
+  joystick_drivers:
+    doc:
+      type: git
+      url: https://github.com/ros2/joystick_drivers.git
+      version: master
```
Comment (clalancette) [inline crystal/distribution.yaml]: This should be ros2

File: crystal/distribution.yaml [hunk 2]
```diff
@@ -396,6 +396,24 @@ repositories:
...
+      - joy
+      tags:
+        release: release/crystal/{package}/{version}
+      url: https://github.com/ros2-gbp/joystick_drivers-release.git
+      version: 2.2.0-1
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/ros2/joystick_drivers.git
+      version: master
```
Comment (clalancette) [inline crystal/distribution.yaml]: Ditto.

**Review by clalancette — CHANGES_REQUESTED** [review]
doc and source versions should both be `ros2` (I may have messed this up in my previous PR as well).

File: crystal/distribution.yaml (hunk 1 above)
Comment (nuclearsandwich) [reply crystal/distribution.yaml]: This was definitely me entering them. But please do check the previous ones.

**Review by clalancette — APPROVED** [review]

**Pull request merged**